corehq/apps/callcenter/data_source.py

```diff
@@ -75,7 +77,7 @@ def _make_data_source_for_domain(data_source_json, domain_name):
     doc = deepcopy(data_source_json)
     doc['domain'] = domain_name
     doc['_id'] = StaticDataSourceConfiguration.get_doc_id(domain_name, doc['table_id'])
-    return DataSourceConfiguration.wrap(doc)
+    return _wrap_data_source_by_doc_type(doc)
```

Comment:
probably not necessary here but also doesn't hurt

Reply:
It looks like this was causing some problems, specifically a `KeyError` on `doc_type`.